src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AssessmentService.kt

```diff
@@ -74,7 +74,6 @@ class AssessmentService(
   private val domainEventService: DomainEventService,
   private val offenderService: OffenderService,
   private val communityApiClient: CommunityApiClient,
-  private val cruService: CruService,
```

Comment:
I think CruService can be deleted now? 🤞 that will make a net-negative line count PR 😄